src/gui/guiScene.cpp

```diff
@@ -23,6 +23,7 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include <SViewFrustum.h>
 #include <IAnimatedMeshSceneNode.h>
 #include <ILightSceneNode.h>
+#include "irrlichttypes.h"
```

Comment:
How that could fix it since `irrlichttypes_extrabloated.h` is already included in the header? And it failed again.